t/nvmept_fdp.py

```diff
@@ -674,6 +725,72 @@ def check_all_ruhs(dut):
         "test_class": FDPTest,
         "success": SUCCESS_NONZERO,
     },
+    # Specify invalid options related to dataplacement scheme
+    ## using old and new sets of options
+    {
+        "test_id": 302,
+        "fio_opts": {
+            "rw": 'write',
+            "bs": 4096,
+            "io_size": 4096,
+            "verify": "crc32c",
+            "fdp": 1,
+            "fdp_pli": 3,
+            "fdp_pli_select": "scheme",
+            "output-format": "normal",
+        },
+        "test_class": FDPTest,
+        "success": SUCCESS_NONZERO,
+    },
+    {
+        "test_id": 303,
+        "fio_opts": {
+            "rw": 'write',
+            "bs": 4096,
+            "io_size": 4096,
+            "verify": "crc32c",
+            "fdp": 1,
```

Comment:
This test is identical to the previous one.
`"fdp": 1,"` should be `"dataplacement": 1"`
`"fdp_pli": 3` should be `"plids": 3`